website/docs/r/datasync_location_fsx_windows_file_system.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "DataSync"
+layout: "aws"
+page_title: "AWS: aws_datasync_location_fsx_windows_file_system"
+description: |-
+  Manages an FSx Windows Location within AWS DataSync.
+---
+
+# Resource: aws_datasync_location_fsx_windows_file_system
+
+Manages an AWS DataSync FSx Windows Location.
+
+## Example Usage
+
+```hcl
+resource "aws_datasync_location_fsx_windows_file_system" "example" {
```

Comment:
Terraform 0.12 syntax now for examples I think after #8950.

Reply:
oops, i though CI already covers this. fixing.